spaceward/src/features/modals/ApproveActionModal.tsx

```
@@ -11,8 +11,11 @@ import { useModalState } from "./state";
 import clsx from "clsx";
 import { usePublicClient } from "wagmi";
 import { env } from "@/env";
+import { useConnectWallet } from "@web3-onboard/react";
 
 export default function ApproveModal({ hidden }: ModalParams<{}>) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)